### ISSUE TITLE
Console error when using browser autofill

### ISSUE THREAD

**Issue by k-ivan** [jackocnr/intl-tel-input#1586]
### Steps to reproduce
1. Create a small form with 3 fields. Field with name, phone and email. The browser must have this data for autofill. 
2. Accept the browser’s offer to insert data into the fields.

Example: https://codepen.io/k-ivan/pen/VwNqoBm?editors=1010

### Expected behaviour
There are no errors in the console

### Actual behaviour
An error appears in the console. The insert itself works successfully.
```
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at t._handleKeydownEvent (intlTelInput.min.js:16:18929)
```
![image](https://github.com/jackocnr/intl-tel-input/assets/6150895/d77dc55d-e0cf-4b17-9ffa-8f1baa9d232e)

### Initialisation options
```js
  formatAsYouType: true,
  formatOnDisplay: true,
  strictMode: true,
  utilsScript: "https://cdn.jsdelivr.net/npm/intl-tel-input@21.2.6/build/js/utils.js"
```

**Comment by jackocnr** [comment]
Thanks for reporting. Fixed in v21.2.7.

**Comment by k-ivan** [comment]
Great, thanks a lot!
